bigframes/ml/decomposition.py

```diff
@@ -27,6 +27,8 @@
 from bigframes.ml import base, core, globals, utils
 import bigframes.pandas as bpd
 
+_BQML_PARAMS_MAPPING = {"svd_solver": "pcaSolver"}
```

Comment:
Same here.

Reply:
This one isn't a new adding, but moved piece.